packages/sdks/src/components/block/components/block-styles.lite.tsx

```diff
@@ -96,7 +97,34 @@ export default function BlockStyles(props: BlockStylesProps) {
           })
         : '';
 
-      return [largeStylesClass, mediumStylesClass, smallStylesClass].join(' ');
+      const hoverAnimation =
+        processedBlock.animations &&
+        processedBlock.animations.find((item) => item.trigger === 'hover');
+
+      let hoverStylesClass = '';
+      if (hoverAnimation) {
+        const hoverStyles = hoverAnimation.steps?.[1]?.styles || {};
+        hoverStylesClass =
+          createCssClass({
+            className: `${className}:hover`,
+            styles: {
+              ...hoverStyles,
+              transition: `all ${
+                hoverAnimation.duration
+              }s ${camelCaseToKebabCase(hoverAnimation.easing)}`,
+              transitionDelay: hoverAnimation.delay
+                ? `${hoverAnimation.delay}s`
+                : '0s',
+            },
+          }) || '';
+      }
+
+      return [
+        largeStylesClass,
+        mediumStylesClass,
+        smallStylesClass,
+        hoverStylesClass,
+      ].join(' ');
```

Comment:
Why do we need to apply the hover at this point? Isn't the hover animation meant to be applied in the Animator class here:

https://github.com/BuilderIO/builder/blob/c58bc3be910b47037bed8c99ea463cde07dc7d5e/packages/sdks/src/components/block/animator.ts#L159-L195

https://github.com/BuilderIO/builder/blob/c58bc3be910b47037bed8c99ea463cde07dc7d5e/packages/sdks/src/components/block/block.lite.tsx#L173-L186

Reply:
we are filtering out hover animations here so its not getting applied.

I followed this (gen1 code) for reference, we are setting hover animations in every block level: https://github.com/BuilderIO/builder/blob/c58bc3be910b47037bed8c99ea463cde07dc7d5e/packages/react/src/components/builder-block.component.tsx#L176